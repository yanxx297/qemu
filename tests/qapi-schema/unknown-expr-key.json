# FIXME: we should reject an expression with unknown top-level keys
{ 'type': 'bar', 'data': { 'string': 'str'}, 'bogus': { } }
