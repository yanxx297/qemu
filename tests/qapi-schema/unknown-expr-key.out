[OrderedDict([('type', 'bar'), ('data', OrderedDict([('string', 'str')])), ('bogus', OrderedDict())])]
[]
[OrderedDict([('type', 'bar'), ('data', OrderedDict([('string', 'str')])), ('bogus', OrderedDict())])]
